tools/wptrunner/requirements_epiphany.txt

```diff
@@ -0,0 +1,2 @@
+mozprocess == 0.26
+selenium == 3.9.0
```

Comment:
I'm not sure that selenium is required since `ExecutorWebDriver` doesn't use it (some existing files are wrong here). We could probably add `mozprocess` to `requirements.txt` and eliminate a bunch of `requirements_*` files.

Reply:
OK, I was confused by that too, but just copy/pasted from requirements_webkit.txt.

(What is `mozprocess` for?)